internal/setup/openpgp_test.go

```diff
@@ -101,17 +104,7 @@ func (s *S) TestVerifySignature(c *C) {
 		sigs, body, _, err := setup.DecodeClearSigned([]byte(test.clearData))
 		if err == nil {
 			// Verify at least one signature with the set of public keys.
```

Comment:
With the change in name I think this comment is redundant, do you agree?

Reply:
Ya, removed it.